### PR TITLE
Toolboxes can hold normal-size items

### PR DESCRIPTION
**Pull request by 9600bauds**
Some people think toolboxes are still useless even after they were buffed, so there should be no problem with this buff :^)
After all, right now you could just find a backpack easily and carry that around instead of the toolbox

### CODE REVIEW

**Comment by jknpj** [comment]
Does that means we can stuff bombs inside of them?

**Comment by 9600bauds** [comment]
Sure why not

**Comment by clusterfack** [comment]
I'm noticing a significant lack of ROBUSTNESS with these toolboxes 
https://github.com/d3athrow/vgstation13/pull/9637

**Comment by JustSumBody** [comment]
Jesus christ that commit description

**Comment by kol1th** [comment]
nice commit friend :+1:

**Comment by gbasood** [comment]
good thing Github doesnt read our repo or we might get shut down for that tbh

**Comment by icantthinkofanameritenow** [comment]
...Did you just copypaste the entire script of Shrek 2 into that changelog summary?

I'm pretty sure that qualifies as copyright infringement, so hurry up and fix it before the DreamWorks lawyers show up or something.

**Comment by ghost** [comment]
@icantthinkofanameritenow

**Comment by icantthinkofanameritenow** [comment]
@DrCelt 
You absolute madman, there's nothing more dangerous than an enraged copyright lawyer.

**Comment by Exxion** [comment]
Bauds is a walking example of why branch names and commit descriptions should have character limits

**Comment by ghost** [comment]
@icantthinkofanameritenow aye but we have literally got copyrighted songs in our roundstart lobby

**Comment by 9600bauds** [comment]
There's literally 3 google pages full of mirrors for that script so I think we're safe in this one boys, but if you insist I'll change it to something else

**Comment by Exxion** [comment]
Nobody who gives a shit is ever going to see it
It's just really long

**Comment by AutismSpider** [comment]
use the bee movie script then

**Comment by ComicIronic** [comment]
Fucking
Time
Cube

But seriously make a proper commit

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 1]
```diff
@@ -17,9 +17,10 @@
 	melt_temperature = MELTPOINT_STEEL
 	origin_tech = "combat=1"
 	attack_verb = list("robusts", "batters", "staves in")
+	fits_max_w_class = W_CLASS_MEDIUM
 	storage_slots = 14
 	max_combined_w_class = 28
-	fits_ignoring_w_class = list(
+	/*fits_ignoring_w_class = list(
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/storage/toolbox.dm:23]: I assume you checked every one of these items to see if it fits?

Comment (9600bauds) [inline code/game/objects/items/weapons/storage/toolbox.dm:23]: Yes

**Comment by 9600bauds** [comment]
That's what squash and merge is for, Comic

**Pull request merged**